src/components/input/input.scss

```diff
@@ -0,0 +1,99 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/motion.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/focus";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Browsers apply certain default styles unless appearance is set to `none`.
+ *    Unfortunately Chrome, Safari and Mozilla all still rely on the
+ *    vendor-prefixed version of this property at the time of this writing.
+ * 2. Input heights vary between different types in certain browsers unless a
+ *    height is explicitly set.
+ * 3. Non-normal line heights break placeholder text alignment in Safari.
+ */
+
+.c-input {
+  -moz-appearance: none; /* stylelint-disable-line property-no-vendor-prefix */
+  -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix */
+  appearance: none; /* 1 */
+  background-color: colors.$text-light;
+  border: sizes.$edge-medium solid currentColor;
+  border-radius: sizes.$radius-medium;
+  color: colors.$text-dark;
+  display: block;
+  font: inherit;
+  font-style: normal;
+  height: sizes.$control-height; /* 2 */
+  line-height: normal; /* 3 */
+  margin: 0;
+  outline: none;
+  padding: sizes.$control-pad-vertical sizes.$control-text-inset;
+  transition-duration: motion.$speed-quick;
+  transition-property: background-color, border-color;
+  transition-timing-function: motion.$ease-out;
+  width: 100%;
+
+  /**
+   * For certain single-line input types, using `text-indent` instead of
+   * `padding-left` gives a more natural and intuitive appearance when text
+   * content overflows.
```

Comment:
Nit: Not sure I get this -- testing with or without the override on an email input I can't really see the difference, but I well may be missing it.

Reply:
With `padding-left`, overflowing text gets awkwardly cut off on the left:

![padding](https://user-images.githubusercontent.com/69633/78281233-384f6100-74cf-11ea-8a18-3b0dff368d3a.gif)

With `text-indent`, it does not:

![text-indent](https://user-images.githubusercontent.com/69633/78281244-3c7b7e80-74cf-11ea-8acb-e5ad98e993ea.gif)

Reply:
Ahh thanks. I didn't fully understand what this was doing either. Adding that sentence to the comment may be helpful

Reply:
> With padding-left, overflowing text gets awkwardly cut off on the left:

Thanks, I missed that. This could be in the comment, too.